sigulsign_unsigned.py

```diff
@@ -89,7 +92,7 @@ def __init__(self, arch=None):
             self.kojihub = "koji"
         self.koji_module = koji.get_profile_module(self.kojihub)
         self.kojisession = self.koji_module.ClientSession(self.koji_module.config.server, {'krb_rdns': False})
-        self.kojisession.ssl_login(CLIENTCERT, CLIENTCA, SERVERCA)
+        self.kojisession.ssl_login(CLIENTCERT, None, None)
```

Comment:
Now that the serverca and clientca are uneeded, you can use self.kojisession.ssl_login(CLIENTCERT) instead.